packages/eslint-plugin-next/lib/rules/image-component/no-unsized-images.js

```diff
@@ -0,0 +1,29 @@
+const createImageRule = require('../../utils/imageComponentRule.js')
+
+module.exports = {
+  meta: {
+    messages: {
+      unsizedImages:
+        'For layout stability, the image component should be used ' +
+        'with a height and width property, even if actual image size is determined with CSS.' +
+        'if you cannot provide a correct-ratio height and width, use the "unsized" attribute.' +
+        'More info: https://web.dev/optimize-cls/#images-without-dimensions',
+    },
+  },
+  create: createImageRule((context, node) => {
+    if (!imageValidSize(node)) {
+      context.report({
+        node,
+        messageId: 'unsizedImages',
+      })
+    }
+  }),
+}
+
+function imageValidSize(node) {
+  let attributes = node.attributes.map((attribute) => attribute.name.name)
+  return (
+    attributes.includes('unsized') ||
+    (attributes.includes('height') && attributes.includes('width'))
```

Comment:
We need to check both name and value of the attribute to avoid passing `<Img height="" width="" ... />`

I think adding a util to extract name and value or return an object with the attribute names and values would be useful, since we do the same parsing inn our other rules too. WDYT?

Reply:
Yep, good idea--I added a util for that.